addons/l10n_bd/data/account.account.tag.csv

```diff
@@ -0,0 +1,4 @@
+"id","name","country_id/id"
+"account_tags_tax_c_exempt","Corporate tax-exempt","base.bd"
+"account_tags_tax_c_cog","Corporate tax of cogs account","base.bd"
+"account_tags_tax_c_exp","Corporate tax of expense accounts","base.bd"
```

Comment:
```suggestion
"id","name"
"account_tags_tax_c_exempt","Bangladesh Corporate Tax - Exempt Income"
"account_tags_tax_c_cog","Bangladesh Corporate Tax - Cost of Goods Sold"
"account_tags_tax_c_exp","Bangladesh Corporate Tax - Expense"
```
Unfortunately the `country_id` field only has an impact on tax tags (these are tags with `applicability='taxes'`, which are set on journal items to denote which line of the tax report they affect), not account tags.
(see the definition of the field in `account_account_tag.py`

So keeping in mind that those tags are available across all companies, so we need to be specific in their name which country they apply to.